client/payment-methods/index.js

```diff
@@ -38,35 +35,19 @@ import ConfirmPaymentMethodActivationModal from './activation-modal';
 import ConfirmPaymentMethodDeleteModal from './delete-modal';
 import { getPaymentMethodDescription } from 'wcpay/utils/payment-methods';
 import CapabilityRequestNotice from './capability-request';
-import InlineNotice from 'wcpay/components/inline-notice';
 import { BuildMissingCurrenciesTooltipMessage } from 'wcpay/components/currency-information-for-methods';
 
 const PaymentMethodsDropdownMenu = ( { setOpenModal } ) => {
-	const { isUpeEnabled, upeType } = useContext( WcPayUpeContext );
-	const isDisablePossible = isUpeEnabled && upeType !== 'deferred_intent';
-	const label = isDisablePossible
-		? __( 'Add feedback or disable', 'woocommerce-payments' )
-		: __( 'Add feedback', 'woocommerce-payments' );
-
-	const buttons = [
-		{
-			title: __( 'Provide feedback', 'woocommerce-payments' ),
-			onClick: () => setOpenModal( 'survey' ),
-		},
-	];
-
-	if ( isDisablePossible ) {
-		buttons.push( {
-			title: 'Disable',
-			onClick: () => setOpenModal( 'disable' ),
```

Comment:
Prior to these changes, there could have been 2 modal "types" available: `disable` and `survey`.
After these changes, there's only going to be one modal "type": `survey`.

Do you think it would be worth it to refactor this to have a boolean to identify whether the modal is open or not?

I also wonder if as part of this cleanup effort we should remove the dropdown menu as a whole, as recommended here: paJDYF-9xf-p2#comment-20835

Reply:
Holding off on all parts of this until we get confirmation: p1702298060113889-slack-C043JN7AJHY
Ok to handle everything separately, given the timing.

Reply:
As discussed, we'll confirm some details p1702298060113889-slack-C043JN7AJHY and get back to this thread afterward.